server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/DownloadGet.feature

```diff
@@ -14,3 +14,9 @@ Feature: Download GET
   Scenario: Getting an attachment with an unknown blobId
     When getting the attachment with an unknown blobId
     Then the user should receive a not found response
+
+  Scenario: Getting an attachment previously stored with a desired name
```

Comment:
could you please work on top of https://github.com/linagora/james-project/pull/294 ?